src/librustdoc/clean/utils.rs

```diff
@@ -109,13 +109,14 @@ pub fn get_deprecation(cx: &DocContext<'_>, def_id: DefId) -> Option<Deprecation
     cx.tcx.lookup_deprecation(def_id).clean(cx)
 }
 
-pub fn external_generic_args(
+fn external_generic_args(
     cx: &DocContext<'_>,
     trait_did: Option<DefId>,
     has_self: bool,
     bindings: Vec<TypeBinding>,
     substs: SubstsRef<'_>,
 ) -> GenericArgs {
+    debug!("cleaning bindings {:?} for external trait {:?}", bindings, trait_did);
```

Comment:
Is it really necessary to add this `debug`?

Reply:
Ok, I removed it.